packages/orchestration/src/utils/zcf-tools.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @import {HostInterface, HostOf} from '@agoric/async-flow';
+ * @import {VowTools} from '@agoric/vow';
+ * @import {ZcfTools} from '../types.js';
+ */
+
+import { M, mustMatch } from '@endo/patterns';
+
+const HandlerShape = M.remotable('OfferHandler');
+
+/**
+ * @param {ZCF} zcf
+ * @param {VowTools} vowTools
+ */
+export const makeZcfTools = (zcf, vowTools) => {
+  /** @satisfies {HostInterface<ZcfTools>} */
+  const zcfForFlows = harden({
```

Comment:
vestigal name.

```suggestion
  const zcfTools = harden({
```
or
```suggestion
  return harden({
```

Reply:
good catch! I thought I caught all of the renames, but clearly not.

I think the const decl is worth keeping cuz `return` doesn't work with `@satisfies`